packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -593,6 +596,35 @@ export class AngularCompilerPlugin {
     }
   }
 
+  private _warnOnUnusedFiles(compilation: compilation.Compilation) {
```

Comment:
It looks like this warning will be shown on all builds, including rebuilds. Is this intentional? If so, we should also test on rebuilds to make sure it works correctly on adding and removing ts files.

Reply:
That was the idea, to show on all builds. What do you think? 

If you agree, I will add some more tests to cover the mentioned test cases.